### ISSUE TITLE
Trying to access array offset on value of type null

### ISSUE THREAD

**Issue by HeJiaNong** [Qoraiche/laravel-mail-editor#65]
When I create a mailables project following the official website example, I will be prompted with an error when I open it：

![image](https://user-images.githubusercontent.com/39476157/72590600-12d3a200-3939-11ea-8403-750487e270f6.png)

[View public sharing errors](https://flareapp.io/share/OmVXrA78)

Then I found that there were no `from` and `replyTo` keys in the $resource array of the blade template. When I changed the `build` method of the mail class, it seemed to be back to normal again

![image](https://user-images.githubusercontent.com/39476157/72590709-5cbc8800-3939-11ea-89f7-4a67dd793165.png)

![image](https://user-images.githubusercontent.com/39476157/72591379-ee78c500-393a-11ea-9866-c46c2b84afd2.png)

This is a new laravel project. I don't know what caused the error

**Comment by base64bits** [comment]
Thanks for creating this awesome package, great work! 

Im facing this same issue, new Laravel project 6.2 and php v7.4

**Comment by 4n70w4** [comment]
The same. PHP 7.4.3, laravel/framework v6.6.1

Trying to access array offset on value of type null (View: /var/www/app/vendor/qoraiche/laravel-mail-editor/resources/views/sections/view-mailable.blade.php)`
![image](https://user-images.githubusercontent.com/38257723/75783234-46478000-5d71-11ea-8ea8-e8688e7dcc2c.png)

**Comment by 4n70w4** [comment]
Workaround:

```patch
Index: vendor/qoraiche/laravel-mail-editor/resources/views/sections/view-mailable.blade.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- vendor/qoraiche/laravel-mail-editor/resources/views/sections/view-mailable.blade.php	(date 1583245647740)
+++ vendor/qoraiche/laravel-mail-editor/resources/views/sections/view-mailable.blade.php	(date 1583245647740)
@@ -52,32 +52,12 @@
 
 				    				<tr>
 	                                    <td class="table-fit font-weight-sixhundred">From</td>
-	                                    <td><a href="mailto:{{ collect($resource['data']->from)->first()['address'] }}{{ !collect($resource['data']->from)->isEmpty() ? collect($resource['data']->from)->first()['address'] : config('mail.from.address') }}" class="badge badge-info mr-1 font-weight-light">
-	                                    	@if (!collect($resource['data']->from)->isEmpty())
-
-                            					{{ collect($resource['data']->from)->first()['address'] }}
-
-                            					@else
-
-												{{ config('mail.from.address') }} (default)
-
-                            				@endif
-                        				</a></td>
+	                                    <td></td>
                                 	</tr>
 
                                 	<tr>
 	                                    <td class="table-fit font-weight-sixhundred">Reply To</td>
-	                                    <td><a href="mailto:{{ collect($resource['data']->replyTo)->first()['address'] }}{{ !collect($resource['data']->replyTo)->isEmpty() ? collect($resource['data']->replyTo)->first()['address'] : config('mail.reply_to.address') }}" class="badge badge-info mr-1 font-weight-light">
-	                                    	@if (!collect($resource['data']->replyTo)->isEmpty())
-
-                            					{{ collect($resource['data']->replyTo)->first()['address'] }}
-
-                            					@else
-
-												{{ config('mail.reply_to.address') }} (default)
-
-                            				@endif
-                        				</a></td>
+	                                    <td></td>
 
                                 	</tr>
```

**Comment by 4n70w4** [comment]
Fixed after upgrading to 1.3.3 and higher
